### ISSUE TITLE
[JSON] JobStateUpdateHandler decodes tuples and dict with int

### ISSUE THREAD

**Issue by chaen** [DIRACGrid/DIRAC#4382]
* [ ] service JobStateUpdateHandler method setJobParameters
* [ ] service JobStateUpdateHandler method setJobsParameter

```
============================================= Decoding tuples =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 777, in __bootstrap
    self.__bootstrap_inner()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 804, in __bootstrap_inner
    self.run()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadPool.py", line 112, in run
    oJob.process()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadPool.py", line 162, in process
    self.__jobResult = self.__jobFunction( *self.__jobArgs, **self.__jobKwArgs )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 367, in _processInThread
    result = self._processProposal(trid, proposalTuple, handlerObj)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 541, in _processProposal
    result = self._executeAction(trid, proposalTuple, handlerObj)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 561, in _executeAction
    return handlerObj._rh_executeAction(proposalTuple)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RequestHandler.py", line 124, in _rh_executeAction
    retVal = self.__doRPC(actionTuple[1])

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RequestHandler.py", line 235, in __doRPC
    retVal = self.__trPool.receive(self.__trid)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/TransportPool.py", line 99, in receive
    received = self.__transports[ trid ][0].receiveData( maxBufferSize, blockAfterKeepAlive, idleReceive )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 253, in receiveData
    data = DEncode.decode(data)[0]

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 499, in decode
    return g_dDecodeFunctions[data[0]](data, 0)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 472, in decodeDict
    oD[k], i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 409, in decodeList
    ob, i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 409, in decodeList
    ob, i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 434, in decodeTuple
    printDebugCallstack('Decoding tuples')

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 253)
RPC call service JobStateUpdateHandler method setJobParameters
With arguments  {'data': 'ds2:OKb1s5:Valueli5elts4:par1s9:par1Valueets4:par2s9:par2Valueeeee'}
```
```
============================================= Decoding dict with numeric keys =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 777, in __bootstrap
    self.__bootstrap_inner()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 804, in __bootstrap_inner
    self.run()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadPool.py", line 112, in run
    oJob.process()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadPool.py", line 162, in process
    self.__jobResult = self.__jobFunction( *self.__jobArgs, **self.__jobKwArgs )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 367, in _processInThread
    result = self._processProposal(trid, proposalTuple, handlerObj)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 541, in _processProposal
    result = self._executeAction(trid, proposalTuple, handlerObj)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Service.py", line 561, in _executeAction
    return handlerObj._rh_executeAction(proposalTuple)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RequestHandler.py", line 124, in _rh_executeAction
    retVal = self.__doRPC(actionTuple[1])

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RequestHandler.py", line 235, in __doRPC
    retVal = self.__trPool.receive(self.__trid)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/TransportPool.py", line 99, in receive
    received = self.__transports[ trid ][0].receiveData( maxBufferSize, blockAfterKeepAlive, idleReceive )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 253, in receiveData
    data = DEncode.decode(data)[0]

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 499, in decode
    return g_dDecodeFunctions[data[0]](data, 0)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 472, in decodeDict
    oD[k], i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 409, in decodeList
    ob, i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 469, in decodeDict
    printDebugCallstack("Decoding dict with numeric keys")

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 253)
RPC call service JobStateUpdateHandler method setJobsParameter
With arguments  {'data': 'ds2:OKb1s5:Valueldi15els10:SomeStatuss7:Runningeeee'}
```

**Comment by fstagni** [comment]
In PR soon.
